include/devicetree/zephyr.h

```diff
@@ -47,10 +40,6 @@
 
 #endif /* DT_DOXYGEN */
 
-#if DT_NODE_HAS_PROP(DT_CHOSEN(zephyr_entropy), label)
-#define DT_CHOSEN_ZEPHYR_ENTROPY_LABEL DT_LABEL(DT_CHOSEN(zephyr_entropy))
-#endif
```

Comment:
This needs to go through the stable API change process.

Reply:
https://github.com/zephyrproject-rtos/zephyr/issues/43230